script/run_glean_baseline_migrate_first_seen

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+## Updates glean usage queries to utilize first seen columns
+: << EOF
```

Comment:
Is this a pattern for making a multiline comment? So the below is just an example invocation?

Reply:
Yes, `:` is a nop that is fed a heredoc, which effectively makes this inert. I think it will show up with `set -x`, so it's not entirely a comment.